packages/cactus-plugin-ledger-connector-besu/package.json

```diff
@@ -86,6 +86,8 @@
     "@hyperledger/cactus-common": "0.5.0",
     "@hyperledger/cactus-core": "0.5.0",
     "@hyperledger/cactus-core-api": "0.5.0",
+    "@hyperledger/cactus-cmd-api-server": "0.5.0",
```

Comment:
@AzaharaC You cannot depend on the API server in the plugin packages (would lead to circular dependencies).